src/examples/accordions/timeline/TimelineDefault.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Timeline } from '@zendeskgarden/react-accordions';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Span } from '@zendeskgarden/react-typography';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={5}>
+      <Timeline>
```

Comment:
Isn't the most recent activity displayed on top, like I saw [here](https://zendeskgarden.github.io/react-components/?path=/story/components-accordions-timeline--default)? 🤔

Reply:
oh geez, maybe you're right. I'm not used to reading a timeline in the upwards direction (a la GitHub). @steue what is the typical up/down timeline for Zendesk?

Reply:
tbh, even if it is typically upward, I feel like the content on the website reads weird in isolation and should be ordered from top to  bottom.

Reply:
I feel like whatever way we're using it in the product (or want it to be used in the product), we should show it that way here, so that if product managers reference this page, they don't get confused.

Reply:
+1 to bethany’s suggestion!

our product’s timeline patterns sorts by top to bottom / most recent to latest. it makes the most sense as agents would need to see most recent / relevant activity. i had a similar slight confusion when looking at the example as i was used to seeing those activities ordered in instructive steps, i.e. locate, arrange, shop, plant. however, chronologically from most recent to latest, plant, shop, arrange, is how i should expect to see it.